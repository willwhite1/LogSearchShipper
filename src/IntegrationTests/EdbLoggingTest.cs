using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using NUnit.Framework;

namespace IntegrationTests
{
	[TestFixture]
	class EdbLoggingTest : IntegrationTestBase
	{
		void Init()
		{
			Init("LogsearchShipper.exe.config.EdbLoggingTest");
		}

		[Test]
		public void TestEdbLogging()
		{
			Init();
		}

		public override string TestName
		{
			get { return "LogSearchShipper.EdbLoggingTest"; }
		}
	}
}
