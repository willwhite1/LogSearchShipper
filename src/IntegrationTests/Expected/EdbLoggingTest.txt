{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\PKH-PPE-APP10\\logs\\Apps\\PriceHistoryService\\log.log","expected_state":"stopped","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\PKH-PPE-APP10\\logs\\Apps\\PriceHistoryService\\log.log","expected_state":"stopped","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\PKH-PPE-APP10\\","expected_state":"stopped","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"PKH-ENV2-SHARED01","service":"PKH-ENV2-SHARED01","event_source":"","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\ENV1-APP01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"PKH-ENV2-SHARED01","service":"PKH-ENV2-SHARED01","event_source":"\\\\PKH-ENV2-SHARED01\\Logs\\Nolio\\all.log","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"DB","host":"ENV1-DB01","service":"ENV1-DB01","event_source":"\\\\ENV1-DB01\\Logs\\Nolio\\all.log","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\ENV1-APP01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\PKH-PPE-APP10\\logs\\Apps\\PriceHistoryService\\PriceHistoryStats.log","expected_state":"stopped","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"DB","host":"ENV1-DB01","service":"ENV1-DB01","event_source":"\\\\ENV1-DB01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"DB","host":"ENV1-DB01","service":"ENV1-DB01","event_source":"\\\\ENV1-DB01\\Logs\\Nolio\\all.log","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"PKH-ENV2-SHARED01","service":"PKH-ENV2-SHARED01","event_source":"","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"PKH-ENV2-SHARED01","service":"PKH-ENV2-SHARED01","event_source":"","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\ENV1-APP01\\Logs\\u*.log","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"DB","host":"ENV1-DB01","service":"ENV1-DB01","event_source":"\\\\ENV1-DB01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"PKH-ENV2-SHARED01","service":"PKH-ENV2-SHARED01","event_source":"\\\\PKH-ENV2-SHARED01\\Logs\\Nolio\\all.log","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\PKH-PPE-APP10\\logs\\Apps\\PriceHistoryService\\PriceHistoryStats.log","expected_state":"stopped","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"DB","host":"ENV1-DB01","service":"ENV1-DB01","event_source":"\\\\ENV1-DB01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\PKH-PPE-APP10\\","expected_state":"stopped","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"DB","host":"ENV1-DB01","service":"ENV1-DB01","event_source":"\\\\ENV1-DB01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\ENV1-APP01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\ENV1-APP01\\","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"ENV1-APP01","service":"ENV1-APP01","event_source":"\\\\ENV1-APP01\\Logs\\u*.log","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
{"environment":"ENV1","cluster":"APP","host":"PKH-ENV2-SHARED01","service":"PKH-ENV2-SHARED01","event_source":"","expected_state":"running","logger":"EdbExpectedEventSourcesLogger","level":"INFO"}
